alertrules.yaml

```diff
@@ -0,0 +1,53 @@
+---
+# Downloads Prometheus and Loki alert rules
+
+- name: "Create Prometheus and Loki alert rules bundle"
+  hosts: localhost
```

Comment:
This shouldn't be localhost unless the target host is the one running the playbook. This could be a `prometheus_agent` role or something along those lines, then you designate which hosts to run these tasks on in your inventory. Additionally, any reason this isn't part of the main entrypoint? i.e site.yaml

Reply:
the target host is the one running the playbook. These files must be copied to Prometheus/Loki servers, which are not managed by Ansible.